app/views/events/index.html.erb

```diff
@@ -0,0 +1,65 @@
+<% if user_signed_in? %>
+<div>
+    <h2 class="text-center text-danger my-5">Your Events</h2>
+    <div class="d-flex flex-column justify-content-center align-items-center">
+        <% current_user.events.each do |event| %>
+            <ul>
+                <p class="text-center"><strong class="text-primary"><%= event.title %></strong></p>
```

Comment:
#### MANDATORY
- [ ] To list events created by user, i.e. `@created_events = current_user.events`, kindly define this as an instance variable in your controller and use the variable in your view. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/current_user_show_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))